File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbDatabase.java
package com.alvinhkh.buseta.nlb.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class NlbDatabase {

    @SerializedName("version")
    public String version;

    @SerializedName("routes")
    public List<NlbRoute> routes;

    @SerializedName("stop_districts")
    public List<NlbStopDistrict> stop_districts;

    @SerializedName("stops")
    public List<NlbStop> stops;

    @SerializedName("route_stops")
    public List<NlbRouteStop> route_stops;

    public String toString() {
        return "NlbDatabase{version=" + this.version + "}";
    }

}

File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbDatabase.kt
package com.alvinhkh.buseta.nlb.model

import com.google.gson.annotations.SerializedName

data class NlbDatabase(
        @SerializedName("version")
        var version: String = "",

        @SerializedName("routes")
        var routes: List<NlbRoute> = arrayListOf(),

        @SerializedName("stop_districts")
        var stop_districts: List<NlbStopDistrict> = arrayListOf(),

        @SerializedName("stops")
        var stops: List<NlbStop> = arrayListOf(),

        @SerializedName("route_stops")
        var route_stops: List<NlbRouteStop> = arrayListOf()
)
